instance/tests/models/test_instance_mixins.py

```diff
@@ -381,9 +381,11 @@ def check_mysql(self):
         databases = subprocess.check_output("mysql -u root -e 'SHOW DATABASES'", shell=True).decode()
         for database in self.instance.mysql_database_names:
             self.assertIn(database, databases)
-            mysql_cmd = "mysql -u {user} --password={password} -e 'SHOW TABLES' {db_name}".format(
-                user=self.instance.mysql_user,
+            # Pass password using MYSQ_PWD environment variable rather than the --password
+            # parameter so that mysql command doesn't print a security warning.
+            mysql_cmd = "MYSQL_PWD={password} mysql -u {user} -e 'SHOW TABLES' {db_name}".format(
```

Comment:
`subprocess.call` allows you to pass `env` dictionary, you could pass variable there. This would increase security as to read environmental variables of another process you need to be an administrative user (AFAIK).